### ISSUE TITLE
Issue with sitemap detail route

### ISSUE THREAD

**Issue by jlachancekffein** [studioespresso/craft-seo-fields#15]
This rules is not working correctly : 
```
'sitemap_<siteId:\d>_<type:(entry|product|category)>_<sectionId:\d>_<handle:.*>.xml' => 'seo-fields/sitemap/detail'
```

> src/craftCMS/vendor/studioespresso/craft-seo-fields/src/SeoFields.php (line 216)

Working : http://.../sitemap_1_entry_4_about_single.xml
Not working : http://.../sitemap_1_entry_10_careers_channel.xml

The problem is the (:\d) regex symbol. You need to use (:\d+) because siteId and sectionid param can have more than one digit.

I fix it by overriding the rules inside our project module.

```
$event->rules = array_merge($event->rules, [
    'sitemap_<siteId:\d+>_<type:(entry|product|category)>_<sectionId:\d+>_<handle:.*>.xml' => 'seo-fields/sitemap/detail'
]);
```

But it's not ideal to have to override this plugin route.

Can you fix it please ?

**Comment by janhenckens** [comment]
Yes, I'll put out a release tonight!

**Comment by janhenckens** [comment]
Fixed 1.0.3.
